scenes/quests/lore_quests/quest_001/2_ink_combat/ink_combat_round_3.tscn

```diff
@@ -31,7 +31,7 @@ resource_name = "goal_completed"
 tracks/0/type = "method"
 tracks/0/imported = false
 tracks/0/enabled = true
-tracks/0/path = NodePath("%CollectibleItem")
+tracks/0/path = NodePath("../OnTheGround/CollectibleItem")
```

Comment:
I guess this wasn't a change you deliberately made but I don't think we want to replace the unique name reference (`%`) with a relative path to the node.

Reply:
Yep let's not commit those. You can choose which parts of a file to commit while adding, using: `git add --patch` or just `git add -p`.